rpm/remove-dependency.patch

```diff
@@ -0,0 +1,10 @@
+--- setup.py.bak	2017-10-24 21:30:48.620552756 -0400
++++ setup.py	2017-10-24 21:43:59.691747057 -0400
+@@ -27,7 +27,6 @@
+     name='Pext',
+     version=version,
+     install_requires=[
+-        'pyqt5'
+     ],
```

Comment:
Does this patch even apply cleanly? Because my setup.py also contains a pygit2 line before pyqt5, which GitHub doesn't seem to be displaying in this patch.